src/renderer/item.store.ts

```diff
@@ -155,7 +155,7 @@ export abstract class ItemStore<T extends ItemObject = ItemObject> {
 
   async removeSelectedItems?(): Promise<any>;
 
-  subscribe(...args: any[]) {
+  subscribe(..._args: any[]) {
```

Comment:
just remove `...args` is fine.

Reply:
Even if someone wants to overload this function? I think a follow up PR making this an optional function would be better.